scss/_forms.scss

```diff
@@ -215,6 +215,8 @@ textarea.form-control {
   &[disabled] ~ .form-check-label {
     color: $text-muted;
   }
+
+  @include form-control-focus($ignore-warning: true);
 }
```

Comment:
tried adding a commit to this directly, but seems it may not be accepting commits from maintainers...

in any case, I'd suggest adding the following here to mitigate the slightly unsightly look of round radio buttons with a completely square box shadow in IE/Edge (Firefox already rounds them, and Chrome is just stubborn about any styling of checkboxes/radio buttons in general)

```
  &[type="radio"] {
    // Note: This has no effect on the appearance of the radio button, but does smooth out the box shadow on focus in most browsers.
    @include border-radius($input-border-radius, 0);
  }
```

Reply:
I made [the change](https://github.com/alienlebarge/bootstrap/commit/565177310f939c6d452971649c115d18edf07f18) on the [add-check-input-focus-style-v4](https://github.com/alienlebarge/bootstrap/tree/add-check-input-focus-style-v4) branch on [alienlebarge/bootstrap](https://github.com/alienlebarge/bootstrap) but it doesn't appear in the PR...

Do you have any idea why ?

Reply:
@patrickhlauke done